en/SUMMARY.md

```diff
@@ -138,15 +138,18 @@
       * [Hex Here2](gps_compass/rtk_gps_hex_here2.md)
   * [Telemetry Radios](telemetry/README.md)
     * [SiK Radio](telemetry/sik_radio.md)
-      * [RFD900 (SiK) Telemetry Radio](telemetry/rfd900_telemetry.md)
-      * [HKPilot (SIK) Telemetry Radio](telemetry/hkpilot_sik_radio.md)
-      * [HolyBro (SIK) Telemetry Radio](telemetry/holybro_sik_radio.md)
+      * [RFD900 \(SiK\) Telemetry Radio](telemetry/rfd900_telemetry.md)
+      * [HKPilot \(SIK\) Telemetry Radio](telemetry/hkpilot_sik_radio.md)
+      * [HolyBro \(SIK\) Telemetry Radio](telemetry/holybro_sik_radio.md)
     * [Telemetry Wifi](telemetry/telemetry_wifi.md)
       * [ESP8266 WiFi Module](telemetry/esp8266_wifi_module.md)
-      * [3DR Telemetry Wifi (Discontinued)](telemetry/3dr_telemetry_wifi.md)
-  * [MAVLink Telemetry (OSD/GCS)](peripherals/mavlink_peripherals.md)
+      * [3DR Telemetry Wifi \(Discontinued\)](telemetry/3dr_telemetry_wifi.md)
+  * [MAVLink Telemetry \(OSD/GCS\)](peripherals/mavlink_peripherals.md)
   * [FrSky Telemetry](peripherals/frsky_telemetry.md)
-  * [Distance Sensors (Rangefinders)](sensor/rangefinders.md)
+  * [Power module](power_module/REAME.md)
```

Comment:
Note, filename and title changes to include cuav.

Reply:
@cuhome This one still not fixed. The filename should be changed to README.md both here, and in the actual filename